packages/editor/api-report.api.md

```diff
@@ -2735,6 +2735,7 @@ export abstract class StateNode implements Partial<TLEventHandlers> {
     getCurrentToolIdMask(): string | undefined;
     getIsActive(): boolean;
     getPath(): string;
+    getShouldShowIndicators(): boolean;
```

Comment:
For similar flags in e.g. shape utils we drop the `get` and just do `canDoX()` etc.

For things like this where the function name is unambiguously returning a boolean and not performing an action, I think we can drop the `get` and have a shorter, easier to read name